ubi8-openjdk-21.yaml

```diff
@@ -0,0 +1,56 @@
+# This is an Image descriptor for Cekit
+
+schema_version: 1
+
+from: "registry.access.redhat.com/ubi8/ubi-minimal"
+name: &name "ubi8/openjdk-21"
+version: &version "1.18"
+description: "Source To Image (S2I) image for Red Hat OpenShift providing OpenJDK 21"
+
+labels:
+- name: "io.k8s.description"
+  value: "Platform for building and running plain Java applications (fat-jar and flat classpath)"
+- name: "io.k8s.display-name"
+  value: "Java Applications"
+- name: "io.openshift.tags"
+  value: "builder,java"
+- name: "maintainer"
+  value: "Red Hat OpenJDK <openjdk@redhat.com>"
+- name: "com.redhat.component"
+  value: "openjdk-21-ubi8-container"
+- name: "usage"
+  value: "https://access.redhat.com/documentation/en-us/red_hat_jboss_middleware_for_openshift/3/html/red_hat_java_s2i_for_openshift/"
```

Comment:
Should this point to?

https://jboss-container-images.github.io/openjdk/

Reply:
Thank you yes: resolved in 4eff5d8d6293b84de7221c2b8551e122390fcf0b.